### DISCUSSION TITLE
Minor P4 fixes to support SSL encrypted case-sensitive server, and special characters

### DISCUSSION THREAD

**Discussion by 437569** [belkiss/niftyplugins#28] [Ideas]
Hi,

Thanks for maintaining an easy to use VS P4 plugin. I have used your fork both for VS 2019 and 2022 and like it a lot.

I do still have to make some minor fixes so that it can:
1. Support SSL encrypted P4 server, and P4 proxy setup
2. Support filename with special characters ('%', '#' and '@'). (Yes in some game engine those characters are unavoidable somewhat ... 😞 )

I open pull request #26 and #27 to share what I have done to fix 2. and 1. respectively. In #26 I also took out extra p4 info calls when invoking p4 command, since I notice it's already passing the cwd, using which p4 command already applies the same p4 set. The p4 info call can be slow on proxy setup. Please feel free to merge them if you think they are acceptable.

Salute!

**Comment by belkiss** [comment]
Thanks for the kind words! I integrated #27, and waiting for your input about handling '*' (I guess it would be even worse to have that in filenames!).

Cheers!

**Reply by 437569** [reply]
Thank you so much for the great work. Can't wait for the next plugin release ❤️

**Reply by belkiss** [reply]
Will run with it locally for a bit, and if all's well I'll make a release, probably sometime next week :)

Thanks again!
